hedera-node/src/test/java/com/hedera/services/legacy/unit/handler/SmartContractRequestHandlerPayableTest.java

```diff
@@ -237,12 +237,12 @@ public void createContractWithAdminKey() {
     TransactionRecord record = smartHandler.createContract(body, consensusTime, contractBytes, seqNumber);
     ledger.commit();
 
-    Assert.assertNotNull(record);
-    Assert.assertNotNull(record.getTransactionID());
-    Assert.assertNotNull(record.getReceipt());
-    Assert.assertEquals(ResponseCodeEnum.SUCCESS, record.getReceipt().getStatus());
-    Assert.assertEquals(contractSequenceNumber, record.getReceipt().getContractID().getContractNum());
-    Assert.assertTrue(record.hasContractCreateResult());
+    Assertions.assertNotNull(record);
+    Assertions.assertNotNull(record.getTransactionID());
+    Assertions.assertNotNull(record.getReceipt());
+    Assertions.assertEquals(ResponseCodeEnum.SUCCESS, record.getReceipt().getStatus());
+    Assertions.assertEquals(contractSequenceNumber, record.getReceipt().getContractID().getContractNum());
+    Assertions.assertTrue(record.hasContractCreateResult());
```

Comment:
nit: for these and all following `Assertions.assert*` below,  might be easier to read and less redundant if they are static imported.

Reply:
Agreed. Would also highly recommend assert4j to clean it up and chain assertions:
```java
assertThat(record)
    .isNotNull()
    .returns(true, TransactionRecord::hasContractResult)
    .returns(transactionId, TransactionRecord:: getTransactionID)
    .extracting(TransactionRecord::getReceipt)
    .returns(ResponseCodeEnum.SUCCESS, TransactionReceipt::getStatus)
    .returns(contractSequenceNumber, r -> r.getContractID().getContractNum());
```

Reply:
@ljianghedera, I was doing a global find/replace and didn't want to invest further as this is a legacy test code.
@steven-sheehy, good point, `assertThat` would definitely improve readability. We will consider this for writing new tests or refactoring old ones.
Thanks both!